### PR TITLE
feat: adds the 'Social Engineering' library

### PR DESCRIPTION
**Pull request by Coppini**
Adds `Skills`, `Advantages` and `Modifiers` from [Social Engineering](http://www.sjgames.com/gurps/books/socialengineering/).

Resolves https://github.com/richardwilkes/gcs_master_library/issues/110

### CODE REVIEW

File: Library/Social Engineering/Social Engineering Modifiers.adm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	"rows": [
+		{
+			"type": "modifier",
+			"id": "39ce3558-7397-40b5-8cf8-1d51b7efb9da",
+			"name": "Aspected",
+			"reference": "SE77",
+			"cost_type": "percentage",
+			"cost": -20,
+			"affects": "total",
+			"notes": "Choose aspect"
```
Comment (richardwilkes) [inline Library/Social Engineering/Social Engineering Modifiers.adm:14]: Typically this is done using replaceable text, i.e. something like "@Aspect@", which will then prompt the user to enter in a value when they try to apply it to a character sheet. You can put these in the name or notes field.

**Review by richardwilkes — COMMENTED** [review]
Generally looks good. Had one comment about the possible use of replaceable text, which may apply elsewhere as well -- I didn't look closely enough to identify other possible spots, though.

File: Library/Social Engineering/Social Engineering Modifiers.adm (hunk 1 above)
Comment (richardwilkes) [reply Library/Social Engineering/Social Engineering Modifiers.adm:14]: @Coppini were you planning to update this?

Comment (Coppini) [reply Library/Social Engineering/Social Engineering Modifiers.adm:14]:
```suggestion
			"notes": "@Aspect@"
```
Sorry, hadn't noticed your comment before.

Comment (Coppini) [reply Library/Social Engineering/Social Engineering Modifiers.adm]: Updated 👍🏻

**Review by richardwilkes — APPROVED** [review]

**Pull request merged**